src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const priceRent = days * 40;
+
+  if (days < 3) {
+    return priceRent;
+  } else if (days >= 3 && days < 7) {
+    return priceRent - 20;
+  } else if (days >= 7) {
+    return priceRent - 50;
+  }
```

Comment:
```suggestion
  if (days < 3) {
    return priceRent;
  } 
   if (days < 7 ) {
    return priceRent - 20;
  } 
    return priceRent - 50;
  }
```
Best practice ---> вынеси дискаунт в переменные